inference-engine/src/transformations/src/transformations/common_optimizations/fq_reshape_fusion.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/fq_reshape_fusion.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+#include <numeric>
+
+ngraph::pass::FakeQuantizeReshapeFusion::FakeQuantizeReshapeFusion() {
+    const auto fq_node_p = ngraph::pattern::wrap_type<opset4::FakeQuantize>(
+            {ngraph::pattern::wrap_type<opset4::Constant>(), // for weights only
+             ngraph::pattern::any_input(),
+             ngraph::pattern::any_input(),
+             ngraph::pattern::any_input(),
+             ngraph::pattern::any_input()},
+            pattern::consumers_count(1));
+    const auto reshape_node_p = ngraph::pattern::wrap_type<opset4::Reshape>(
+            {fq_node_p, ngraph::pattern::any_input()}, pattern::consumers_count(1));
```

Comment:
Why you check consumers here? Is it really needed?

Reply:
Nope. Copy-paste problems, sorry.